### PR TITLE
adds image sizes to pistachio

### PR DESCRIPTION
**Pull request by leejordan**

### CODE REVIEW

File: site/docs/views/layouts_feature-block-examples.hbs [hunk 1]
```diff
@@ -9,7 +9,9 @@
             <div class="grid__col-xs-12">
                 <h1 class="cd-title" id="banner">Feature block examples</h1>
                 <p class="cd-text">This page outlines all the possible layouts for feature blocks. To see a full page laid out using these feature blocks, check out the <a href="/layouts/example-feature-page">example feature page</a>.</p>
+                <p class="cd-text">All images should be saved from photoshop via the "save for web" option at a suggested jpg quality size of 30.</p>
```
Comment (notlee) [inline site/docs/views/layouts_feature-block-examples.hbs]: A little prescriptive, and there are better tools, but better than not having any kind of optimisation reminder. 👍

File: site/docs/views/layouts_feature-block-examples.hbs [hunk 2]
```diff
@@ -9,7 +9,9 @@
...
                 <h2 class="cd-title">Feature banner</h2>
+                <h3 class="cd-title">image size: 1200px x 300px</h3>
```
Comment (notlee) [inline site/docs/views/layouts_feature-block-examples.hbs]: Could we at least addd 'suggested image size'? Size depends on how it's used on-site which Pistachio doesn't prescribe. Couldn't I add a full bleed feature banner not in a container?

File: site/docs/views/layouts_feature-block-examples.hbs (hunk 1 above)
Comment (leejordan) [inline site/docs/views/layouts_feature-block-examples.hbs]: Do I need to reply to pass this review?

File: site/docs/views/layouts_feature-block-examples.hbs (hunk 2 above)
Comment (leejordan) [inline site/docs/views/layouts_feature-block-examples.hbs]: Not really, the banner is designed to work with that exact size image and we would need to add a modifier or new component for a full width banner.

Also, comms team just require clear guidance and I think saying suggested would just introduce ambiguity.

**Pull request merged**